packages/mui-material/src/Autocomplete/Autocomplete.d.ts

```diff
@@ -226,13 +243,13 @@ export interface AutocompleteProps<
    *
    * @param {T[]} value The `value` provided to the component.
    * @param {function} getTagProps A tag props getter.
-   * @param {object} autocompleteProps The props of the Autocomplete component.
+   * @param {object} state The state of the Autocomplete component.
    * @returns {ReactNode}
    */
   renderTags?: (
     value: T[],
     getTagProps: AutocompleteRenderGetTagProps,
-    autocompleteProps: AutocompleteProps<T, Multiple, DisableClearable, FreeSolo, ChipComponent>,
+    state: AutocompleteOwnerState<T, Multiple, DisableClearable, FreeSolo, ChipComponent>,
```

Comment:
Call it `ownerState` please, as this name is used across the codebase and it'll be clear what state we mention.

Reply:
Done 👍